### PR TITLE
update index and test html files

### PR DESCRIPTION
**Pull request by Roland-Sankara**
Update the UI of the applications.
Changes are made in the index and test HTML files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,14 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="stylesheet" href="style.css" />
-  <style>
-    span {
-      color: purple
-    }
-  </style>
-  <title>Edutech Teens</title>
+  
+  <title>KanzuCodeFundation</title>
```
Comment (Lucas-Masaba) [inline index.html:10]: - Awesome title

File: index.html [hunk 2]
```diff
@@ -6,16 +6,14 @@
...
 </head>
 
 <body>
 
+  <h1>KanzuCodeFundation is awesome!</h1>
+
```
Comment (Lucas-Masaba) [inline index.html:16]: - It's alright

**Review by Lucas-Masaba — APPROVED** [review]
- The changes are perfect.

File: index.html (hunk 2 above)
Comment (Roland-Sankara) [reply index.html:16]: OMG

Comment (Roland-Sankara) [reply index.html:16]: Thanks

**Pull request merged**